std_msgs/Empty request
---
Location goal
